cosmos-gui/src/mysql_driver.js

```diff
@@ -37,89 +37,99 @@ var connection = mysql.createConnection({
     database: mysqlConfig.database
 });
 
-function connect(callback) {
-    connection.connect(function (error) {
+function addUser(idm_username, username, password, hdfsQuota, callback) {
+    pool.getConnection(function(error, connection) {
         if (error) {
             callback(error);
```

Comment:
Since you are exporting the function `addUser()` which "may" take a callback (the "client" calling it may pass it or not), maybe you would like to protect the calling of the callback with something such as:
```javascript
if (callback) {
  callback();
}
```